docs/hazmat/primitives/twofactor.rst

```diff
@@ -171,3 +180,34 @@ similar to the following code.
         :param int time: The time value to validate against.
         :raises cryptography.hazmat.primitives.twofactor.InvalidToken: This
              is raised when the supplied TOTP does not match the expected TOTP.
+
+    .. method:: get_provisioning_uri(account_name, issuer)
+
+        :param str account_name: The display name of account, such as
+            ``'Alice Smith'`` or ``'alice@example.com'``.
+        :param issuer: The optional display name of issuer.
+        :type issuer: `string` or `None`
+        :return str: An URI string.
+
+Provisioning URI
+~~~~~~~~~~~~~~~~
+
+The provisioning URI of HOTP and TOTP is not actual the part of RFC 4226 and
+RFC 6238, but a `spec of Google Authenticator <http://git.io/vkvvY>`_. It is
```

Comment:
Our typical model for external links is to put them at the bottom of the page like:

```
.. _`spec of Google Authenticator`: https://github.com/google/google-authenticator/wiki/Key-Uri-Format
```

Additionally we strongly prefer not to use shortener links where possible so the canonical URL is preferred.